src/main/java/org/truffleruby/language/globals/WriteSimpleGlobalVariableNode.java

```diff
@@ -0,0 +1,52 @@
+package org.truffleruby.language.globals;
+
+import org.truffleruby.core.basicobject.BasicObjectNodes.ReferenceEqualNode;
+import org.truffleruby.language.RubyNode;
+import org.truffleruby.language.objects.shared.WriteBarrierNode;
+
+import com.oracle.truffle.api.dsl.Cached;
+import com.oracle.truffle.api.dsl.NodeChild;
+import com.oracle.truffle.api.dsl.Specialization;
+
+@NodeChild(value = "value")
+public abstract class WriteSimpleGlobalVariableNode extends RubyNode {
```

Comment:
The `Simple` here means no hooks, right?

Reply:
Yes. I decided to use simple as the adjective because there is already the method `isSimple()` method on the storage.